### PR TITLE
Doc: Update attribute for ECS messaging wrt default (VPR) 

### PR DESCRIPTION
**Pull request by karenzone**
Co-authored-by: Ry Biesemeyer <yaauie@users.noreply.github.com>

Companion PR to https://github.com/elastic/logstash/pull/13427

### CODE REVIEW

File: docs/versioned-plugins/include/attributes-ls-vpr.asciidoc [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+
+Text is written to accommodate multiple versions because plugins are not stack versioned.  
 /////
 
-:ecs-default: The `pipeline.ecs_compatibility` setting is available in {logstash-ref}/logstash-settings-file.html[`logstash.yml`] and `pipelines.yml`. Any value in `pipeline.ecs_compatibility` other than `disabled` is considered BETA and unsupported until Logstash 8.0 and the final 7.x version are released. Having this pipeline level flag set will cause even patch-level upgrades to automatically consume breaking changes in the upgraded plugins, changing the shape of data the plugin produces.
+
+:ecs-default: When the `ecs_compatibility` option for this plugin is not explicitly set, its effective value depends on the `pipeline.ecs_compatibility` setting for the pipeline in `pipelines.yml`, or globally in {logstash-ref}/logstash-settings-file.html[`logstash.yml`], allowing you to specify your preferred behavior at the plugin, pipeline, or system level. If no preference is specified, the default value is `v8` for Logstash 8 or `disabled` for all earlier releases of Logstash. For more information about ECS compatibility settings in Logstash and plugins, see {logstash-ref}/ls-ecs.
```
Comment (karenzone) [inline docs/versioned-plugins/include/attributes-ls-vpr.asciidoc:21]:
```suggestion
:ecs-default: When the `ecs_compatibility` option for this plugin is not explicitly set, its effective value depends on the `pipeline.ecs_compatibility` setting for the pipeline in `pipelines.yml`, or globally in {logstash-ref}/logstash-settings-file.html[`logstash.yml`], allowing you to specify your preferred behavior at the plugin, pipeline, or system level. If no preference is specified, the default value is `v8` for Logstash 8 or `disabled` for all earlier releases of Logstash. For more information about ECS compatibility settings in Logstash and plugins, see {logstash-ref}/ls-ecs[ECS in Logstash].
```

**Comment by karenzone** [comment]
UPDATE: Switched status to DRAFT pending resolution of `master` path issue documented in LSR test PR: https://github.com/elastic/logstash-docs/pull/1218.
>The {logstash-ref} attribute used in the default attribute is looking for the master branch, which has been renamed to main. ToDo: Find out why and fix it.

**Review by karenzone — COMMENTED** [review]
DO NOT MERGE until content is finalized for LSR and reflected here.

**Comment by karenzone** [comment]
@yaauie  Review notes: See #1222 for a head start on testing/validation. I started with this PR, and then applied the variable to a test file.  I added a link to the preview in the test PR.

**Review by yaauie — APPROVED** [review]
LGTM 🎅🏼

**Pull request merged**